plugins/repository-s3/src/test/resources/rest-api-spec/test/repository_s3/10_basic.yml

```diff
@@ -11,3 +11,186 @@
         nodes.info: {}
 
     - match:  { nodes.$master.plugins.0.name: repository-s3  }
+---
+"Snapshot/Restore with repository-s3":
+  - skip:
+      version: " - 6.3.99"
+      reason: repository-s3 was not testable through YAML tests until 6.4.0
```

Comment:
Is this because fixture won't be running? In that case you shouldn't need this because the fixture will always be running if you have this change.

Reply:
Good to know, thanks!